src/ucp/wireup/address.c

```diff
@@ -1653,6 +1602,25 @@ static ucp_rsc_index_t ucp_address_get_remote_device_index(
     return ucs_array_length(device_array) - 1;
 }
 
+static void
+ucp_address_unpacked_adjust_legacy(ucp_unpacked_address_t *unpacked_address)
+{
+    uint8_t legacy_minor_api_ver = ucp_address_packed_release_version(18);
+    size_t i;
+
+    if ((unpacked_address->addr_version != UCP_OBJECT_VERSION_V1) ||
+        (unpacked_address->dst_version >
+         ucp_address_unpack_release_version(legacy_minor_api_ver))) {
```

Comment:
why not just compare to 18 (like in other places in the code)?

Reply:
to compare apples to apples, which other places do you mean?

Reply:
I'm not sure how it will work in future...

Reply:
per discussion agreed that better to compare vs raw version